validator/src/main.rs

```diff
@@ -1058,6 +1058,17 @@ pub fn main() {
                 .help("IP address for the node to advertise in gossip when \
                       --entrypoint is not provided [default: 127.0.0.1]"),
         )
+        .arg(
+            Arg::with_name("gossip_rpc_addr")
+                .long("gossip-rpc-address")
```

Comment:
`--advertised-rpc-address` or `--public-rpc-address`?   `--gossip-rpc-address` reads a little hard for me ("why does gossip need an RPC address?")

Reply:
Switched to `--public-rpc-address`